_site/assets/sass/vha/pages/home copy/masthead/_index.scss

```diff
@@ -0,0 +1,202 @@
+@use "uswds-core" as *;
+
+.masthead {
+    background: linear-gradient(235deg, #167db9, #4c599e, #00abe6);
+    background-size: 600% 600%;
+    // height: auto;
+    position: relative;
+    
+    -webkit-animation: GradientRotation 40s ease infinite;
+    -moz-animation: GradientRotation 40s ease infinite;
+    animation: GradientRotation 40s ease infinite;
+
+    @include at-media(mobile) {
+        // padding-top: 78%;
+        height: 81vw;
+        margin-bottom: 16%;
+    }
+
+    @include at-media(mobile-lg) {
+        // padding-top: 78%;
+        height: 75vw;
+        margin-bottom: 16%;
+    }
+
+    @include at-media(tablet) {
+        // padding-top: 70%;
+        height: 75vw;
+        margin-bottom: 14%;
+    }
+
+    @include at-media(tablet-lg) {
+        // padding-top: 59%;
+        height: 60vw;
+        margin-bottom: 10%;
+    }
+
+    @include at-media(desktop) {
+        // padding-top: 55%;
+        height: 56vw;
+        margin-bottom: 9%;
+    }    
+
+    @include at-media(desktop-lg) {
+        // padding-top: 54%;
+        height: 54vw;
+        margin-bottom: 8%;
+    }
+
+    @include at-media(widescreen) {
+        height: 730px;
+        padding-top: 0;
+        margin-bottom: 120px;
+    }
+
+    &__content-wrapper {
+        position: absolute;
+        top: 0; 
+        left: 0;
+        width: 100%;
+        padding-top: 48px;
+        z-index: 10;
+    }
+
+    &__logo {
+        margin-bottom: 1.5rem;
+
+        img {
+            height: 78px;
+            
+            @include at-media(tablet) {
+                height: 114px;
+            }
+            
+        }
+
+    }
+
+    &__tagline {
+        font-size: 1.375em;
+        margin-bottom: 48px;
+        color: #fff;
+    }
+
+    &__border {
+        position: absolute;
+        bottom: 0;
+        left: 0; 
+        width: 100%;
+        z-index: 1;
+        background-color: #167DB9;
+        height: 14px;
+        justify-content: end;
+        display: flex;
+
+        .block {
+            position: relative;
+            width: 80px;
+            height: 14px;
+            transform: skew(-40deg, 0deg);
+
```

Comment:
we can add on line 100 to fill in the corner:
transform-origin: bottom left;

![Screenshot 2023-09-13 at 11 35 52 AM](https://github.com/WBDynamics/VA-DOE/assets/137827146/db19a29a-a59b-4530-8af7-0fa5e53fe0eb)